src/DistortableImageOverlay.js

```diff
@@ -228,13 +228,13 @@ L.DistortableImageOverlay = L.ImageOverlay.extend({
   },
 
   setCorners(latlngObj) {
-    var map = this._map;
-    var zoom = map.getZoom();
-    var edit = this.editing;
-    var i = 0;
+    const map = this._map;
+    const zoom = map.getZoom();
+    const edit = this.editing;
+    let i = 0;
 
     // this is to fix https://github.com/publiclab/Leaflet.DistortableImage/issues/402
-    for (var k in latlngObj) {
+    for (let k in latlngObj) {
```

Comment:
Since we use `let` here, we need to also declare `k` with `let` on line 246.

Same thing applies to line 280

Reply:
Okay, i'll get it done

Reply:
Done! @TildaDares